python/whylogs/api/store/utils/random_strings.py

```diff
@@ -0,0 +1,7 @@
+import random
+import string
+
+
+def random_string(length: int = 7):
+    letters = string.ascii_lowercase
```

Comment:
do the filenames need to be readable and short?

I'm concerned this could generate inappropriate/offensive filenames, and if this is not for human readability, then I'd prefer something like a uuid string here instead.

Reply:
done!